packages/core/scripts/local/DeployEMP.js

```diff
@@ -26,9 +26,13 @@ const { interfaceName, ZERO_ADDRESS, parseFixed } = require("@uma/common");
 const { getAbi, getTruffleContract } = require("../../index");
 const argv = require("minimist")(process.argv.slice(), {
   boolean: ["test"],
-  string: ["identifier", "collateral", "cversion"]
+  string: ["identifier", "collateral", "cversion", "name", "symbol", "expiry"]
 });
 const abiVersion = argv.cversion || "1.2.2"; // Default to most recent mainnet deployment, 1.2.2.
+const syntheticName = argv.name || "Test Synth";
+const syntheticSymbol = argv.symbol || "SYNTH";
+const expiry = argv.expiry || 2;
```

Comment:
```suggestion
const expiry = argv.expiry || 2; // Default to expire 2 mins from deployment
```

Reply:
what units are these in? seconds or minutes

Reply:
Good point, these should be minutes.